src/store/creator/reducer.ts

```diff
@@ -15,6 +15,7 @@ export default createReducer<CreatorState>(initialState, builder =>
     builder
         .addCase(createAnnotationAction.fulfilled, state => {
             state.error = null;
+            state.message = '';
```

Comment:
Couldn't you just return the initial state?

Reply:
> Couldn't you just return the initial state?

Good idea